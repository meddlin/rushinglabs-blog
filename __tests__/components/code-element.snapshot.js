import { render } from '@testing-library/react';
import CodeElement from '../../components/content-helpers/code-element';

it('renders CodeElement unchanged', () => {
    const { container } = render(
        <CodeElement>this is a test</CodeElement>
    );

    expect(container).toMatchSnapshot()
});