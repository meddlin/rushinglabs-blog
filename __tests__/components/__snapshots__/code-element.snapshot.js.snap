// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CodeElement unchanged 1`] = `
<div>
  <code
    class=" text-xs bg-slate-200 rounded px-px "
  >
    this is a test
  </code>
</div>
`;
